consensus/spos/bls/sovereignSubRoundEndOutGoingTxData.go

```diff
@@ -43,8 +43,18 @@ func (sr *sovereignSubRoundEndOutGoingTxData) VerifyAggregatedSignatures(bitmap
 }
 
 // AggregateAndSetSignatures aggregates and sets signatures for outgoing tx data
-func (sr *sovereignSubRoundEndOutGoingTxData) AggregateAndSetSignatures(bitmap []byte, epoch uint32) ([]byte, error) {
-	sig, err := sr.signingHandler.AggregateSigs(bitmap, epoch)
+func (sr *sovereignSubRoundEndOutGoingTxData) AggregateAndSetSignatures(bitmap []byte, header data.HeaderHandler) ([]byte, error) {
```

Comment:
is it necessary to have all of these files in the spos/bls? they do not seem to implement SubRoundHandler

Reply:
Yes, better to keep them all together.

Reply:
ok